proxy/core/src/main/scala/io/cloudstate/proxy/ProtobufGeneratedMessageRegisterFeature.scala

```diff
@@ -23,10 +23,20 @@ final class ProtobufGeneratedMessageRegisterFeature extends Feature {
     for {
```

Comment:
The TCK passes using a native-image produced with this Protobuf AutomaticFeature—note that I had to filter specific package names to add methods for, as otherwise it would make the build take forever to run for some reason :S (most likely ballooning memory requirement).